newPathologyView.php

```diff
@@ -6,18 +6,22 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Pharmabase :: New Pathology</title>
+    <title>COJO :: Nouveau Employé</title>
     <?php include "inc/resources.php" ?>
 </head>
 <body>
     <div class="wrapper">
         <header>
-            <h1>Pharmabase</h1>
-            <?php breadcrumb("New Pathology") ?>
+            <h1>COJO</h1>
+            <?php breadcrumb("Nouveau Employé") ?>
         </header>
 
         <form action="doNewPathology.php" method="POST">
-            Name: <input type="text" name="name"><br />
+            Nom: <input type="text" name="name"><br />
+            Prénom: <input type="text" name="name"><br />
+            Adresse permanente: <input type="text" name="name"><br />
+            Adresse olympique: <input type="text" name="name"><br />
+            Téléphone: <input type="text" name="name"><br />
```

Comment:
Ici, le ```name``` doit etre egale au attributs utiliser dans notre database, 

Par example, Prénom: ```<input type="text" name="Prénom"><br />```, et meme chose pour les 4 autres variables.

Please make sure to be consistent with the attributs used in our database